packages/bits/spec/components/menu/menu.atom.ts

```diff
@@ -36,6 +40,7 @@ export class MenuAtom extends Atom {
 
     public getMenuItemByIndex = (idx: number): MenuItemAtom => new MenuItemAtom(this.getItemElements().get(idx));
 
+    // DO NOT USE. The following method is broken and will be fixed in the scope of NUI-6105
```

Comment:
It may be a good idea to change this comment to a jsdoc-style comment so it will show up in an IDE's auto-complete tooltip if someone tries to use it.